lib/index.js

```diff
@@ -14,6 +14,8 @@
       extend = require('./utils/extend'),
       isDateString = require('./utils/assert-date-string');
 
+  handleWindowResize();
```

Comment:
This is part of the initial library setup, and I didn't want it to get lost/forgotten lower in the script, so I put it up near the top. I may move it into the `/lib/libraries/default.js` file, since it's responsible for resizing c3js artifacts.